src/modules/apps/e2e/report/failedApps.tsx

```diff
@@ -80,17 +90,28 @@ const ErrorVisualization: React.FunctionComponent<ErrorVisualizationProps> = ({
   error,
   stack,
   testScreenshots,
+  testVideo,
+  specId,
+  testId,
 }) => {
-  const testScreenshotsText = testScreenshots.map(curScreenshot => ` ${curScreenshot.path}`).join('\n')
+  const { account, workspace } = SessionManager.getSingleton()
+  const testScreenshotsUrl = testScreenshots
+    .map(
+      curScreenshot =>
+        `${workspace}--${account}.myvtex.com/_v/screenshot/${testId}/${specId}/${curScreenshot.screenshotId}`
+    )
+    .join('\n')
+  const videoUrl = testVideo === 'true' ? `${workspace}--${account}.myvtex.com/_v/video/${testId}/${specId}` : 'false'
   return (
     <Box key={title.join('')} flexDirection="column">
       <FailedSpecDetail label="Test" text={title.join(' ')} indented={false} />
       {body && <FailedSpecDetail label="Body" text={body} indented />}
       {stack && <FailedSpecDetail label="Stack" text={stack} indented />}
       {error && <FailedSpecDetail label="Error" text={error} indented />}
       {testScreenshots.length > 0 && (
-        <FailedSpecDetail label="Screenshots" text={testScreenshotsText} indented={testScreenshots.length > 1} />
+        <FailedSpecDetail label="Screenshots" text={testScreenshotsUrl} indented={testScreenshots.length > 1} />
       )}
+      {testVideo && <FailedSpecDetail label="Video" text={videoUrl} indented={false} />}
```

Comment:
If testVideo is 'false' this will still create the video components

testVideo could be converted to boolean, or this could be `testVideo === 'true' && <FailedSpec...`

Reply:
'true'

Reply:
Here `testVideo` would have to be compared against 'true', because in case testVideo os 'false' the component would be rendered anyway